### PR TITLE
Fix two intermittents issues in integration tests

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: test/integration/scripting_spec.js [hunk 1]
```diff
@@ -804,6 +804,10 @@ describe("Interaction", () => {
             window.document.querySelector(selector).scrollIntoView();
           }, getSelector("171R"));
 
+          await page.waitForSelector('[data-page-number="4"][class="page"]', {
```
Comment (Snuffleupagus) [inline test/integration/scripting_spec.js:807]: I'm probably missing something obvious here, but aren't the fields we're filling out below actually on the *second* page of the document?

**Review by Snuffleupagus — APPROVED** [review]

File: test/integration/scripting_spec.js (hunk 1 above)
Comment (calixteman) [reply test/integration/scripting_spec.js:807]: Yes that's correct but some fields are updated on page 4.
Here the goal is to be sure that page 4 is here before trying to scroll it into the view.

**Pull request merged**